integration/tests/karpenter/karpenter_test.go

```diff
@@ -26,34 +28,25 @@ func init() {
 	if err := api.Register(); err != nil {
 		panic(fmt.Errorf("unexpected error registering API scheme: %w", err))
 	}
-	params = tests.NewParams("")
+	params = tests.NewParams("karp")
 }
 
 func TestKarpenter(t *testing.T) {
 	testutils.RegisterAndRun(t)
 }
 
 var _ = Describe("(Integration) Karpenter", func() {
-
-	var (
-		clusterName string
-	)
-
-	BeforeEach(func() {
-		// the randomly generated name we get usually makes one of the resources have a longer than 64 characters name
-		clusterName = fmt.Sprintf("it-karpenter-%d", time.Now().Unix())
-	})
-
 	Context("Creating a cluster with Karpenter", func() {
```

Comment:
Do you want to add back the After to remove the cluster? Otherwise, we'll forget about it, and it will leave some clutter. If it happens to NOT fail, we can always just remove it.

Reply:
oh shit yeah!